contribs/src/main/java/com/netflix/conductor/contribs/kafka/KafkaEventQueueProvider.java

```diff
@@ -22,43 +22,46 @@
 public class KafkaEventQueueProvider implements EventQueueProvider {
     private static final Logger logger = LoggerFactory.getLogger(KafkaEventQueueProvider.class);
     protected Map<String, KafkaObservableQueue> queues = new ConcurrentHashMap<>();
```

Comment:
can queues be final ?

Reply:
Queues are not final, they only have a protected access

Reply:
but they're only initialised once right, regardless of whether they're protected. if so, they can be final unless you're re initialising them in any super class which is not a good idea anyway.

Reply:
Corrected